.circleci/create_circleci_config.py

```diff
@@ -276,6 +276,18 @@ def job_name(self):
 )
 
 
+non_modeling_job = CircleCIJob(
```

Comment:
It's a bit dangerous to do that and I would keep the jobs as they were: sometimes a modification in a non-modeling file will make the library unusable in TensorFlow or Flax because of a missing gated torch import.